### PR TITLE
Front end for request form submission.

### PR DESCRIPTION
**Pull request by gijskant**
Fixes [PODIUM-62](https://jira.thehyve.nl/browse/PODIUM-62).

### CODE REVIEW

File: podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        for (let req of requests) {
+            submittedMessage += `<li>Request ${req.requestDetail.title} for organisation ${req.organisations}.</li>`;
+        }
+        submittedMessage += `</ul>`;
+        this.messageService.store(new Message(submittedTitle, submittedMessage));
+    }
+
+    confirmSubmit (request: RequestBase) {
+        this.requestService.submitDraft(request.uuid).subscribe(response => {
+            this.eventManager.broadcast({ name: 'request',
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts]: Is this used?

File: podium-gateway/src/main/webapp/app/request/form/request-form.route.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { RequestFormComponent } from './request-form.component';
 
 export const requestFormRoute: Route = {
   path: 'new',
+  children: [],
```
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/request/form/request-form.route.ts]: Should not be necessary when route has no children

File: podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import { EventManager, JhiLanguageService } from 'ng-jhipster';
+
+import {RequestService} from '../../shared/request/request.service';
+import {RequestBase} from '../../shared/request/request-base';
+import {MessageService} from '../../shared/message/message.service';
+import {Message} from '../../shared/message/message.model';
+
+@Component({
+    selector: 'request-form-submit-dialog',
+    template: `<form name="submitForm" (ngSubmit)="confirmSubmit(request)">
```
Comment (gijskant) [inline podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts]: Or the other way around, move small templates inline.

File: podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    clear () {
+        this.activeModal.dismiss('cancel');
+    }
+
+    setSubmitSuccessMessage(requests: RequestBase[]) {
+        let submittedTitle = `The request has been successfully submitted.`;
+        let submittedMessage = `<ul>`;
+        for (let req of requests) {
+            submittedMessage += `<li>Request ${req.requestDetail.title} for organisation ${req.organisations}.</li>`;
+        }
```
Comment (rnugraha) [inline podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts]: I would prefer to have this as properties and to have ngFor in template

File: podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts (hunk 3 above)
Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts]: Beforehand you don't know the size of the template when only looking a component.

File: podium-gateway/src/main/webapp/app/request/request.route.ts [hunk 5]
```diff
@@ -15,7 +15,7 @@ import { UserRouteAccessService } from '../shared';
 import { requestFormRoute } from './';
 
 let REQUEST_ROUTES = [
-   requestFormRoute
```
Comment (gijskant) [inline podium-gateway/src/main/webapp/app/request/request.route.ts]: That's what I did

Comment (berndvdveen) [inline podium-gateway/src/main/webapp/app/request/request.route.ts]: Hmm git rendered it incorrectly then. Nevermind

File: podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts (hunk 4 above)
Comment (gijskant) [inline podium-gateway/src/main/webapp/app/request/form/request-form-submit-dialog.component.ts]: Once the 'Completed' component has been refactored, yes, I agree. But that currently takes a String and that's it.

**Pull request merged**